<script>
 try {
   setTimeout(() => {
     throw new Error()
   }, 500);
 } catch (error) {
  console.log(error)

 }
  
</script>